getssl

```diff
@@ -370,35 +370,36 @@ copy_file_to_location() { # copies a file, using scp if required.
   fi
 }
 
+date_direpoc() { # convert the dir date into epoch time
+  if [[ "$os" == "bsd" || "$os" == "mac" ]]; then
+    date -j -f "%F %H:%M" "$1" +%s
+  else
+    date -d "$1" +%s
+  fi
+}
+
 date_epoc() { # convert the date into epoch time
-  if [[ "$os" == "bsd" ]]; then
-    date -j -f "%b %d %T %Y %Z" "$1" +%s
-  elif [[ "$os" == "mac" ]]; then
+  if [[ "$os" == "bsd" || "$os" == "mac" ]]; then # uses older style date function.
     date -j -f "%b %d %T %Y %Z" "$1" +%s
+  elif date --version |& grep -iqw busybox; then # BB needs explicit format
+    date -D "%b %d %T %Y %Z" -d "$1" +%s
```

Comment:
The type check should be moved to get_os function, and the -w option doesn't appear to make sense.